tests/integration/common.rs

```diff
@@ -67,3 +67,32 @@ pub(crate) fn compare_string(a: &String, b: &String) {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::TempDatabase;
+
+    #[test]
+    fn test_statement_columns() -> anyhow::Result<()> {
+        let _ = env_logger::try_init();
+        let tmp_db =
+            TempDatabase::new("create table test (foo integer, bar integer, baz integer);");
+        let conn = tmp_db.connect_limbo();
+
+        let stmt = conn.prepare("select * from test;")?;
+
+        let columns = stmt.columns();
+        assert_eq!(columns.len(), 3);
+        assert_eq!(&columns[0], "foo");
+        assert_eq!(&columns[1], "bar");
+        assert_eq!(&columns[2], "baz");
+
+        let stmt = conn.prepare("select foo, bar from test;")?;
+
+        let columns = stmt.columns();
+        assert_eq!(columns.len(), 2);
+        assert_eq!(&columns[0], "foo");
+        assert_eq!(&columns[1], "bar");
+        Ok(())
+    }
```

Comment:
should we add also tests for cases like
- delete
- insert
- select from 1=1, baz, foo from test
- ...

to have a bit more coverage :)